docs/How-to-use-kaito-in-aks-arc.md

```diff
@@ -0,0 +1,137 @@
+```
+
+## Deploy KAITO via Helm
+1.	Git clone the KAITO repo to your local machine
+2.	Install KAITO operator using command 
+```bash
+helm install workspace ./charts/kaito/workspace --namespace kaito-workspace --create-namespace
+```
+
+## Deploy LLM Model
+<details>
+<summary><b>Explain the Yaml file</b></summary>
+If a user runs Kaito in an on-premise Kubernetes cluster where nodepool auto provision are unavailable, the GPU nodes can be pre-configured.
+
+- the user needs to add the node names in the `preferredNodes` field in the `resource` spec. As a result, the Kaito controller will skip the steps for GPU node provisioning and use the prepared nodes to run the inference workload.
+
+Using the same method user can try all kaito functionalities, example can be found on /examples folder.
+</details>
+
+1.	Create a YAML file with the following template, make sure to replace the placeholders in curly braces with your own information. Please note, the PresetName can be found from the supported model file in KAITO’s github repo.
+```yaml
+apiVersion: kaito.sh/v1alpha1
+kind: Workspace
+metadata:
+  name: { YourDeploymentName }
+resource:
+  instanceType: Standard_NC12s_v3
+  labelSelector:
+    matchLabels:
+      apps: { YourNodeLabel }
+  preferredNodes:
+  - { YourNodeName }
+inference:
+  preset:
+    name: { PresetName }
+
+```
+a sample yaml file can be 
+```yaml
+apiVersion: kaito.sh/v1alpha1
+kind: Workspace
+metadata:
+  name: workspace-falcon-7b
+resource:
+  instanceType: Standard_NC12s_v3
+  labelSelector:
+    matchLabels:
+      apps: falcon-7b
+  preferredNodes: 
+  - moc-lmkq7webq9z
+inference:
+  preset:
+    name: falcon-7b-instruct
+```
+2.	You need to label your GPU node first, `Kubectl label node samplenode app=YourNodeLabel` and then apply the YAML file
+`kubectl apply -f sampleyamlfile.yaml`
+
+ 
+
+## Validate model deployment 
+1.	Validate the workspace using the command `kubectl get workspace`. Please also make sure both `ResourceReady` and `InferenceReady` fields are True before testing with the sample prompt.
+```
+NAME                           INSTANCE           RESOURCEREADY   INFERENCEREADY   JOBSTARTED   WORKSPACESUCCEEDED   AGE
+workspace-falcon-7b            Standard_NC6s_v3   True            True                          True                 4d
+```
+
+2.	You may test the model with a sample prompt: 
+```bash
+export CLUSTERIP=$(kubectl get svc workspace-falcon-7b -o jsonpath="{.spec.clusterIPs[0]}") 
+
+kubectl run -it --rm --restart=Never curl --image=curlimages/curl -- curl -X POST http://$CLUSTERIP/chat -H "accept: application/json" -H "Content-Type: application/json" -d "{\"prompt\":\"<sample_prompt>\"}"
+```
+a sample output will be like
+```bash
+usera@quke-desktop:~/go/src/go.goms.io/aks/kaito$ kubectl run -it --rm --restart=Never curl --image=curlimages/curl -- curl -X POST http://$CLUSTERIP/chat -H "accept: application/json" -H "Content-Type: application/json" -d "{\"prompt\":\"write a poem\"}"

```

Comment:
we should not expose our personal username here, just the command line is enough I think